src/Foundation/AbstractValidator.php

```diff
@@ -38,25 +36,18 @@ public function addConfiguration($callable)
      */
     protected $validator;
 
-    /**
-     * @var Dispatcher
-     */
-    protected $events;
-
     /**
      * @var TranslatorInterface
      */
     protected $translator;
 
     /**
      * @param Factory $validator
-     * @param Dispatcher $events
      * @param TranslatorInterface $translator
      */
-    public function __construct(Factory $validator, Dispatcher $events, TranslatorInterface $translator)
+    public function __construct(Factory $validator, TranslatorInterface $translator)
```

Comment:
That's a tough one... It would be a breaking change and there's not really a graceful way around it, but I don't really see a reason to keep an extra param around, especially such an unnecessary one.

I'd lean towards removing this now, while we're in beta.

Reply:
Yea, there is no way to have some sort of BC layer, we can only directly remove it.

We should probably document it when the time comes, it's a good thing there appears to be no public extensions that would break, but we can't know for closed source code.